starfish/pipeline/decoder/_iss.py

```diff
@@ -0,0 +1,26 @@
+from starfish.decoders import iss
+from ._base import DecoderAlgorithmBase
+
+
+class IssDecoder(DecoderAlgorithmBase):
+    @classmethod
+    def from_cli_args(cls, args):
+        return IssDecoder()
+
+    @classmethod
+    def get_algorithm_name(cls):
+        return "iss"
+
+    @classmethod
+    def add_arguments(cls, group_parser):
+        pass
+
+    def decode(self, encoded, codebook):
+        import pandas
+
+        # TODO this should be loaded from disk
```

Comment:
We should be able to write this toy example to disk, right? I think this probably belongs in a test case, since we expect the codebook to be experiment specific.

Reply:
JK I just read 96. Nice job. 👍